### ISSUE TITLE
[Feature request] Option to merge branches using --no-ff

### ISSUE THREAD

**Issue by ImGamez** [GitFiend/Support#56]
When merging branches it does it using fast-forward, an option to make a merge commit is a must in some cases. (If there is a way to do it, I don't see it 😶)

Cheers ✌.

**Comment by GitFiend** [comment]
Yeah, there currently isn't. I just do what Git does by default. After reading this: https://stackoverflow.com/questions/6701292/git-fast-forward-vs-no-fast-forward-merge I think I understand why you might want an option to choose. I'll have a think about this.
